src/codeu/chat/client/commandline/Chat.java

```diff
@@ -16,6 +16,7 @@
 
 import java.util.List;
 import java.util.Stack;
+import java.util.*;
```

Comment:
Not a big deal, but I'd generally try to avoid wildcard import statements like this and stick with importing specific classes. (Wildcard import statements make it hard to know exactly which classes your class uses.)